ruby/spec/gilded_rose_spec.rb

```diff
@@ -9,12 +9,78 @@
       expect(items[0].name).to eq "foo"
     end
 
-    it "degrades quality over time by one if not at sell by date" do
-      items = [Item.new("foo", 10, 50)]
+    it "degrades quality over time by one if not at sell in date" do
+      items = [Item.new("foo", 10, 30)]
       GildedRose.new(items).update_quality()
-      expect(items[0].quality).to eq 49
+      expect(items[0].quality).to eq 29
+    end
+
+    it "decreases sell in date by one day" do
+      items = [Item.new("foo", 10, 30)]
+      GildedRose.new(items).update_quality()
```

Comment:
Ha, I forget my ruby -- do we need a `()` after update_quality if there are no args?